doc/SNS_Overview.rst

```diff
@@ -0,0 +1,73 @@
+======================
+SNS Overview
+======================
+
+This document summarizes SNS and local RAID discussions in the C2 architecture team. The goal is to provide a starting point for design level discussions and resource estimation. Note that this documents is not a substitute for a list of requirements or use-cases. Intended audience includes T1 team and C2 architects.
```

Comment:
Replace "C2" with Motr.

Replace "Intended audience includes T1 team and C2 architects." with 
```
The intended audience of this document consists of Motr customers, architects, designers and developers
```

Reply:
Incorporated.